contrib/collectd/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "collectd"
+pkgver = "5.12.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+        "--disable-pcie_errors",
+]
+configure_gen = []
+hostmakedepends = [
+        "automake",
+        "bison",
+        "flex",
+        "libtool",
+        "linux-headers",
+        "pkgconf",
+]
+makedepends = [
+        "hiredis-devel",
+        "i2c-tools-devel",
+        "jansson",
+        "libcurl-devel",
+        "libdbi-devel",
+        "libiptc-devel",
+        "libmicrohttpd-devel",
+        "libmnl-devel",
+        "libnotify-devel",
+        "libpcap-devel",
+        "libsensors-devel",
+        "libvirt-devel",
+        "libxml2-devel",
+        "lua5.4-devel",
+        "protobuf-c-devel",
+        "rabbitmq-c-devel",
+        "rrdtool-devel",
+]
+checkdepends = []
+pkgdesc = "System statistics collection daemon"
+maintainer = "Josh Dorsey <jdorsey@netgate.com>"
+license = "MIT"
+url = "https://collectd.org"
+source = f"https://github.com/collectd/collectd/releases/download/{pkgname}-{pkgver}/{pkgname}-{pkgver}.tar.bz2"
```

Comment:
```suggestion
source = f"https://github.com/collectd/collectd/releases/download/collectd-{pkgver}/collectd-{pkgver}.tar.bz2"
```

Reply:
Can I ask for the rationale here?  Only because I noticed contrib/rrdtool uses `{pkgname}` in the same way, and that was the only reason I did it.

Reply:
we used to put {pkgname} in source urls before and now we don't :)

not much else to it

Reply:
gotcha :)